### PR TITLE
Fixes #12354 - obsolete uneeded gems

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
@ehelms  said he tried this and it didn't work, but he couldn't remember why, so retrying now

File: rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -140,6 +140,10 @@ BuildRequires: %{?scl_prefix}rubygem(qpid_messaging) < 0.31.0
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
 
+#old dependencies that are no longer needed, remove in katello 2.6
```
Comment (ehelms) [inline rubygem-katello/rubygem-katello.spec:143]: 2.4

**Comment by ehelms** [comment]
The test will come with upgrading with a scratch build that includes this - that's where it failed for me iirc.

File: rubygem-katello/rubygem-katello.spec (hunk 1 above)
Comment (jlsherrill) [inline rubygem-katello/rubygem-katello.spec:143]: No, i was implying to leave this obsoletes in there for two releases and then remove them.  I could see an argument for just one release though.

**Comment by jlsherrill** [comment]
@ehelms this seems to work fine for me (updated to obsolete all 3 replaced rpms). 

@stbenjam since @ehelms is likely gone, are you okay with these changes?

**Comment by stbenjam** [comment]
Looks ok to me, 2 releases sounds good to make sure we get nearly everyone.

ACK

**Comment by stbenjam** [comment]
Also, hooray, HAML is :skull: dead

**Pull request merged**